src/components/AddPet.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export default function AddPet() {
```

Comment:
I see that document has no 2 spaces indent. Please check it out. (Check other files too)